src/Game.cpp

```diff
@@ -31,6 +31,164 @@
 #include "UnkObjects/unk6C2E88.hpp"
 #include "UnkObjects/unk6C3250.hpp"
 #include "UnkObjects/unk6C3890.hpp"
+#include "UnkObjects/unk8600F8.hpp"
+#include "config.hpp"
+
+static void sub_612A80(char *args) {
+    glo_8600F0 = 0;
+    glo_8600EC = nullptr;
+    char *esi = args;
+    if (*esi != '\0') {
```

Comment:
This `if` (and the `if` below that's basically an `else`) seems redundant to me. Calling `std::strchr` on an empty string (with the char not being the null character) returns nullptr anyways, so dropping this `if` should make sense.